### PR TITLE
Taking Today as props. Help in some extreme case when you wanted to …

### PR DESCRIPTION
**Pull request by nsisodiya**
…show different today than your current timeZone.

### CODE REVIEW

File: src/day.jsx [hunk 1]
```diff
@@ -16,10 +16,17 @@ var Day = React.createClass({
     includeDates: React.PropTypes.array,
     filterDate: React.PropTypes.func,
     selected: React.PropTypes.object,
+	  today: React.PropTypes.object,
```
Comment (rafeememon) [inline src/day.jsx]: How is this prop set?  Does it need to be passed down from the top level?

File: src/day.jsx [hunk 2]
```diff
@@ -16,10 +16,17 @@ var Day = React.createClass({
...
     startDate: React.PropTypes.object,
     endDate: React.PropTypes.object
   },
 
+	getDefaultProps: function() {
```
Comment (rafeememon) [inline src/day.jsx]: Please fix up the indentation =)

**Comment by rafeememon** [comment]
Please also add test cases for the new prop.  Thanks!

File: src/day.jsx (hunk 2 above)
Comment (martijnrusschen) [inline src/day.jsx]: Strange, linter should catch this too.

File: src/day.jsx (hunk 1 above)
Comment (nsisodiya) [inline src/day.jsx]: Its optional. you can pass or you need not to pass. that is why I have added `getDefaultProps`

File: src/day.jsx [hunk 3]
```diff
@@ -16,10 +16,17 @@ var Day = React.createClass({
     includeDates: React.PropTypes.array,
     filterDate: React.PropTypes.func,
     selected: React.PropTypes.object,
+    today: React.PropTypes.object,
```
Comment (nsisodiya) [inline src/day.jsx]: This is Optional.

**Comment by nsisodiya** [comment]
Ok, I am done with Test Cases !!

**Comment by rafeememon** [comment]
Hold off on merging until discusson on #338 is resolved

File: src/day.jsx (hunk 3 above)
Comment (rafeememon) [inline src/day.jsx]: Continuing from before -- it's optional, but how does the consumer use it?  It's not being passed down from the datepicker component.

**Pull request closed without merging**